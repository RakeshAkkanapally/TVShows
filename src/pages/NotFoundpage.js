import React from "react";
import { makeStyles, Typography } from "@material-ui/core";
import { NavLink } from "react-router-dom";

const useStyles = makeStyles({
  root: {
    backgroundColor: "#fff",
    margin: 0,
    padding: 50,
  },
  text: {
    margin: 6,
    paddingLeft: 80,
  },
  image: {
    height: 500,
  },
});

const NotFoundpage = () => {
  const classes = useStyles();

  return (
    <div id="wrapper" className={classes.root}>
      <div id="info" className={classes.text}>
        <h3>
          This page could not be found
          <NavLink exact to="/" activeClassName={classes.activeLink}>
            <Typography className={classes.hover}>Dashbaord</Typography>
          </NavLink>
        </h3>
      </div>
      <img className={classes.image} src="https://i.imgur.com/qIufhof.png" />
    </div>
  );
};

export default NotFoundpage;
