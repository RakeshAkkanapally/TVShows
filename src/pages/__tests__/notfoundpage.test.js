import React from "react";
import { BrowserRouter } from "react-router-dom";
import renderer from "react-test-renderer";
import NotFoundpage from "../NotFoundpage";

it("renders correctly", () => {
  const tree = renderer
    .create(
      <BrowserRouter>
        <NotFoundpage />
      </BrowserRouter>
    )
    .toJSON();
  expect(tree).toMatchSnapshot();
});
