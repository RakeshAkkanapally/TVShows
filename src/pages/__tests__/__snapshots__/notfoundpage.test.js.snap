// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<div
  className="makeStyles-root-1"
  id="wrapper"
>
  <div
    className="makeStyles-text-2"
    id="info"
  >
    <h3>
      This page could not be found
      <a
        aria-current="page"
        className="active"
        href="/"
        onClick={[Function]}
        style={Object {}}
      >
        <p
          className="MuiTypography-root MuiTypography-body1"
        >
          Dashbaord
        </p>
      </a>
    </h3>
  </div>
  <img
    className="makeStyles-image-3"
    src="https://i.imgur.com/qIufhof.png"
  />
</div>
`;
